core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -285,19 +336,8 @@ private Object handleTxWriteCommand(InvocationContext ctx, WriteCommand command,
       return invokeNextInterceptor(ctx, command);
```

Comment:
The comment above doesn't make sense any more, you're not replicating the command here.